src/Messages/TSBlockingManager.m

```diff
@@ -66,6 +66,10 @@ - (instancetype)initStorageManager:(TSStorageManager *)storageManager messageSen
     _storageManager = storageManager;
```

Comment:
nit: I guess I missed this in the last CR, but shouldn't this method be called init**With**StorageManager, since it doesn't initialize **the** storage manager?